### ISSUE TITLE
Check for numeric argument of exit

### ISSUE THREAD

**Issue by ingydotnet** [koalaman/shellcheck#1388]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
if some-rare-condition; then
    exit foo bar
fi
```

#### Here's what shellcheck currently says:

nothing

#### Here's what I wanted or expected to see:

I ran across this bogus exit statement in some code, and was surprised shellcheck didn't flag it. If the code ever got executed, Bash would flag it, but it's in a rarely true if statement.

**Comment by koalaman** [comment]
Well caught. This was a blatant oversight.

`return` already had such warnings, and now `exit` does as well.
